connector/avro/src/main/scala/org/apache/spark/sql/avro/SchemaConverters.scala

```diff
@@ -80,6 +80,8 @@ object SchemaConverters {
       case DOUBLE => SchemaType(DoubleType, nullable = false)
       case FLOAT => SchemaType(FloatType, nullable = false)
       case LONG => avroSchema.getLogicalType match {
+        case d: CustomDecimal =>
+          SchemaType(DecimalType(d.precision, d.scale), nullable = false)
```

Comment:
Why is it not nullable?  Is it a an Avro thing? Other types seem to be the same.

Reply:
I have no idea. Decimal for bytes or fixed has the same nullable value, and we have no reason to be inconstant with Decimal.